bot/seasons/valentines/lovecalculator.py

```diff
@@ -0,0 +1,119 @@
+    '5': '\U0001F499 A small acquaintance \U0001F499',
+    '1': '\U0001F494 There\'s no real connection between you two \U0001F494'
+}
+LOVE_TXT = {
+    '0': "You two will most likely have the perfect relationship. But don't think that this means you don't have to "
+         "do anything for it to work. Talking to each other and spending time together is key, even in a seemingly "
+         "perfect relationship.",
+    '95': "Your relationship will most likely work out perfect. This doesn't mean thought that you don't need to put "
+          "effort into it. Talk to each other, spend time together, and you two wont have a hard time.",
+    '80': "Your relationship will most likely work out well. Don't hesitate on making contact with each other though, "
+          "as your relationship might suffer from a lack of time spent together. Talking with each other and spending "
+          "time together is key.",
+    '60': "Your relationship will most likely work out. It won't be perfect and you two need to spend a lot of time "
+          "together, but if you keep on having contact, the good times in your relationship will overweigh the bad "
+          "ones.",
+    '45': "Your relationship has a reasonable amount of working out. But do not overestimate yourself there. Your "
+          "relationship will suffer good and bad times. Make sure to not let the bad times destroy your relationship, "
+          "so do not hesitate to talk to each other, figure problems out together etc.",
+    '30': "The chance of this relationship working is not very high, but its not that low either. If you both want "
+          "this relationship to work, and put time and effort into it, meaning spending time together, talking to "
+          "each other etc., than nothing shall stand in your way.",
+    '20': "The chance of this relationship working is not very high. You both need to put time and effort into this "
+          "relationship, if you want it to work out well for both of you. Talk with each other about everything and "
+          "don't lock yourself up. Spend time together. This will improve the chances of this relationship's survival "
+          "by a lot",
+    '5': "There might be a chance of this relationship working out somewhat well, but it is not very high. With a lot "
+         "of time and effort you'll get it to work eventually, however don't count on it. It might fall apart quicker "
+         "than you'd expect.",
+    '1': "The chance of this relationship working out is really low. You can get it to work, but with high costs and "
+         "no guarantee of working out. Do not sit back, spend as much time together as possible, talk a lot with each "
+         "other to increase the chances of this relationship's survival "
+}
+
+
+class LoveCalculator:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('love_calculator', 'love_calc'))
```

Comment:
Since this command accepts user mentions, this should probably have a [cooldown](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.cooldown) in order to reduce spamming.

Reply:
Is a cooldown of 5 seconds per User enough or should I raise that?